spring-jdbc/src/test/java/org/springframework/jdbc/object/SqlQueryTests.java

```diff
@@ -168,7 +168,6 @@ void testStringQueryWithResults() throws Exception {
 		given(resultSet.next()).willReturn(true, true, true, false);
 		given(resultSet.getString(1)).willReturn(dbResults[0], dbResults[1], dbResults[2]);
 		StringQuery query = new StringQuery(dataSource, SELECT_FORENAME);
-		query.setRowsExpected(3);
```

Comment:
Please revert this for the time being as well, since we'll remove it in conjunction with #34530.